lib/rspec/request_describer.rb

```diff
@@ -70,5 +70,11 @@ def self.included(base)
         end
       end
     end
+
+    # @private
+    def self.process_with_kwargs?
+      ActionDispatch::Integration::Session.instance_method(:process) ||
+      ActionDispatch::Integration::Session.instance_method(:process_with_kwargs) rescue false
```

Comment:
Oh, it's my bad. I didn't know arguments of `#process` had changed between actionpack 4.X and 5.0.